apps/bluetooth/js/modules/pair_manager.js

```diff
@@ -294,21 +295,24 @@ define(function(require) {
         showPairviewCallback: this.showPairview.bind(this, pairingInfo)
       };
       // Prepare notification toast.
-      var title = _('bluetooth-pairing-request-now-title');
-      var body = pairingInfo.evt.deviceName || _('unnamed-device');
+      var titleL10n = 'bluetooth-pairing-request-now-title';
+      var bodyL10n = pairingInfo.evt.deviceName ? 
+          { raw: pairingInfo.evt.deviceName } :
+          'unnamed-device';
```

Comment:
this file seems to be using 2-space indentation. Please, follow the files indentation patterns.